src/app/app.component.spec.ts

```diff
@@ -118,4 +119,12 @@ describe('AppComponent', () => {
     document.body.click();
     expect(component.settingsOpen).toBe(false);
   });
+
+  it('should update theme when theme changed', () => {
+    TestBed.createComponent(AppComponent);
```

Comment:
The first line of this test shouldn't be necessary, an AppComponent is created in the `beforeEach`